tree/30_generic_methods/service_disabled.cf

```diff
@@ -51,38 +46,34 @@ bundle agent service_disabled(service_name)
 
       "undefine_at_boot"
         usebundle => ncf_services("${service_name}", "disable"),
-        ifvarclass => "(!has_promiser_stack.service_check_disabled_at_boot_${canonified_service_name}_not_ok)|(has_promiser_stack.${class_prefix}_check_disabled_not_ok)";
+        ifvarclass => "service_check_disabled_at_boot_${canonified_service_name}_not_ok";
```

Comment:
we should use the class_prefix, rather than the old_class_prefix, so service_check_disabled_at_boot_${canonified_service_name}_${canonified_service_name}_not_ok

Reply:
As the method takes only one parameter, shouln't it behave exactly the same?

Reply:
it should, but i guess that in the futur we'll drop one of the two, and it'll be either if naming is consistent